vyked/decorators/tcp.py

```diff
@@ -86,6 +86,44 @@ def wrapper(self, *args, **kwargs):
     wrapper.is_request = True
     return wrapper
 
+def deprecated_api(func=None, new_api=""):
```

Comment:
Also would prefer if it was just called 'deprecated' not 'deprecated_api', the api piece is obvious
```python
@deprecated
def get_something(self, thinga_migik):
    pass
```